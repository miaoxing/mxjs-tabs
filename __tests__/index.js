import React from 'react';
import {render} from '@testing-library/react';
import {Tabs} from '..';

describe('test', () => {
  test('basic', () => {
    const result = render(<Tabs/>);
    expect(result.container).toMatchSnapshot();
  });
});

