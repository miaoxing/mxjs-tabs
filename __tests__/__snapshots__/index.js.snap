// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test basic 1`] = `
<div>
  <nav
    class="nav-underline header-nav nav nav-tabs"
    role="tablist"
  />
  <div
    class="tab-content"
  />
</div>
`;
